codecov_cli/commands/upload.py

```diff
@@ -250,6 +269,9 @@ def do_upload(
         coverage_files_search_explicitly_listed_files=list(
             coverage_files_search_explicitly_listed_files
         ),
+        test_result_files_search_explicitly_listed_files=list(
```

Comment:
We can't process the data, so this is effectively doing useless work as of now.
Let's turn it off until we can start ingesting this.

We should probably also feature flag it somehow. I'd start by setting the `disable_test_search` option default to `True`